client/app/lib/components/redux-form/SingleFileInput.jsx

```diff
@@ -21,20 +21,42 @@ const styles = {
     marginRight: 16,
     width: 150,
   },
+  image: {
+    maxWidth: 300,
+    maxHeight: 300,
+    width: 'auto',
+    height: 'auto',
+  },
   badgeIconActive: {
     backgroundColor: grey400,
   },
   badgeIconInactive: {
     backgroundColor: grey300,
   },
-  dropzone: {
+  dropzoneStyle1: {
```

Comment:
Instead of `dropzoneStyle1`, perhaps do something more nested, eg:
```js
  ...,
  dropzone: {
    avatar: {
      ...
    },
    badge: {
      ...
    }, 
  },
  ...,
```